   float32 throttle
   float32 brake
   float32 steering_rate
   float32 steering_target
   bool parking_brake
   float32 speed
   float32 acceleration